index.js

```diff
@@ -33,6 +33,10 @@ function fastifyStatic (fastify, opts, next) {
 
   function pumpSendToReply (request, reply, pathname) {
     const stream = send(request.raw, pathname, sendOptions)
+    let resolvedFilename
```

Comment:
Can you use `var`?

Reply:
I should point out that nothing in this source uses `var`.  On the other hand `checkRootPathForErrors` uses `let`.  Is there a reason `var` would be preferred over `let` in this specific case?

I've updated the README.md as your suggestion is a big improvement, but I'm not sure about using var here.

Reply:
Can you update the other `let`s to `var`?

Reply:
Thank you for explaining the reason, both now use `var`.